content/en/network_monitoring/performance/network_analytics.md

```diff
@@ -20,41 +21,67 @@ further_reading:
 
-{{< img src="network_performance_monitoring/network_page/context_npm2.png" alt="context" style="width:80%;">}}
+{{< img src="network_performance_monitoring/network_analytics/context_npm2.png" alt="context" style="width:80%;">}}
 
 The next example shows all aggregate connections from IP addresses representing services in region `us-east-1` to availability zones:
 
-{{< img src="network_performance_monitoring/network_page/flow_table_region_az2.png" alt="Aggregate connection table filtered" style="width:80%;">}}
+{{< img src="network_performance_monitoring/network_analytics/flow_table_region_az2.png" alt="Aggregate connection table filtered" style="width:80%;">}}
 
 You can set the timeframe over which traffic is aggregated using the time selector at the top right of the page:
 
-{{< img src="network_performance_monitoring/network_page/npm_timeframe.png" alt="Time frame NPM" style="width:30%;">}}
+{{< img src="network_performance_monitoring/network_analytics/npm_timeframe.png" alt="Time frame NPM" style="width:30%;">}}
 
 Tags from Datadog integrations or Unified Service Tagging can be used for aggregating and filtering automatically. See [custom facets](#custom-facets), below, for other tags. You can also select "Auto-grouped traffic" to see traffic bucketed into several commonly used tags such as `service`, `kube_service`, `short_image`, and `container_name`.
 
-You can filter to traffic where the client or server matches a CIDR using `CIDR(network.client.ip, 10.0.0.0/8)` or `CIDR(network.server.ip, 10.0.0.0/8)`  
+You can filter to traffic where the client or server matches a CIDR using `CIDR(network.client.ip, 10.0.0.0/8)` or `CIDR(network.server.ip, 10.0.0.0/8)`.
+
+### Recommended queries
+
+{{< img src="network_performance_monitoring/network_analytics/recommended_query_options.png" alt="The Network Analytics page in Datadog displaying three recommended queries" style="width:80%;">}}
+
+Recommended queries allow you to jump-start your investigation into your network – whether you're troubleshooting a specific issue or gaining a better overall understanding of your network. The recommended queries help you quickly find relevant network information without the need to search for group the traffic. For example, the recommended query `Find dependencies of service: web-store` populates the search bar with the query `client_service: web-store` and displays the top services that the service web-store is sending traffic to within the network, and therefore its downstream dependencies. 
+
+Any available recommended queries are provided at the top of the Analytics page, and there are three recommended queries at the top of the DNS page. Use these queries to access commonly used data, and see any changes in that data in the last hour. 
```

Comment:
If it's not too inconvenient, it may be helpful providing a screenshot of the three queries on the DNS tab to show the difference between the queries on that page vs. what's on the Analytics tab!

Reply:
Thanks for that feedback, I added both a screenshot and link to the DNS page in [1aa019c](https://github.com/DataDog/documentation/pull/19098/commits/1aa019cec9fe21359ea6edc9e25bf341ee176e3f) to compare the options. Do you think it would be better to show users the screenshot, or let them use the link to see the queries themselves?

Reply:
I like it, but I can also see us removing it too—here's an example for what we did with out-of-the-box queries in the [Synthetic Monitoring & Continuous Testing Explorer](https://docs.datadoghq.com/continuous_testing/explorer/?tab=cibatches#create-a-search-query)!